testData/invalid/yaml/sam_20161031_api_methodsettings_missing_required.yaml

```diff
@@ -0,0 +1,83 @@
+AWSTemplateFormatVersion : '2010-09-09'
```

Comment:
Test runners with descriptions should be added to here https://github.com/martysweet/cfn-lint/blob/master/src/test/validatorTest.ts#L1757

Reply:
Sorry for the oversight. Goint to fix it.